homeassistant/components/tts/marytts.py

```diff
@@ -0,0 +1,127 @@
+"""
+Support for the MaryTTS service.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tts/marytts/
```

Comment:
The link is wrong, should be `https://home-assistant.io/components/tts.marytts/`.